release-notes-4.5.0.md

```diff
@@ -1,10 +1,18 @@
-!not-ready-for-release!
-
 #### Version Number
 ${version-number}
 
 #### New Features
-* [SCMOD-10362](https://portal.digitalsafe.net/browse/SCMOD-10362): GraalVM JavaScript support  
-    The Document Worker Framework has been enhanced to support using the [GraalVM](https://www.graalvm.org/) JavaScript engine.  In previous versions of the framework Nashorn was the only JavaScript engine supported.  For backwards compatibility reasons it remains the default JavaScript engine, but note that it has been deprecated (see [JEP372](https://openjdk.java.net/jeps/372)), so it may not be supported by all future versions of Java.
+- [SCMOD-10362](https://portal.digitalsafe.net/browse/SCMOD-10362): GraalVM JavaScript support  
+  The Document Worker Framework has been enhanced to support using the [GraalVM](https://www.graalvm.org/) JavaScript engine.  In previous versions of the framework Nashorn was the only JavaScript engine supported.  For backwards compatibility reasons it remains the default JavaScript engine, but note that it has been deprecated (see [JEP372](https://openjdk.java.net/jeps/372)), so it may not be supported by all future versions of Java.
+- [SCMOD-11069](https://portal.digitalsafe.net/browse/SCMOD-11069): Update archetype to openSUSE Leap 15.2.
+- [SCMOD-11144](https://portal.digitalsafe.net/browse/SCMOD-11144): Upgrade json-schema-core to 1.2.14  
+  This fixes a vulnerability pointed out by [Fortify](https://ams.fortify.com/Releases/269904/Issues/109539445)
+
+#### Bug Fixes
+- [SCMOD-11362](https://portal.digitalsafe.net/browse/SCMOD-11362): Close script context  
```

Comment:
SCMOD-11362 is not a bug fix since it was not a bug that was present in the last release.  I.e. if we had released the GraalVM support sooner, and then had to do a subsequent release to fix it then yes we would list this as an important bug fix.  But since this is only fixing something that we hadn't released yet we shouldn't be listing it.